domains/kekda.json

```diff
@@ -1,12 +1,12 @@
 {
   "description": "A description",
-  "repo": "https://github.com/kekda-py/kekda-py.github.io",
+  "repo": "https://github.com/kekda-py/the-repo-is-private",
   "owner": {
-    "username": "kekda-py",
```

Comment:
Why did you change this? Your username is kekda-py

Reply:
ohh wait...

Reply:
fixed